ods_ci/tests/Resources/CLI/ModelServing/llm.resource

```diff
@@ -857,3 +860,25 @@ Verify Model KServe Deployment
     ${n_containers}=    Get Length    ${containers}
     Run Keyword And Continue On Failure    Verify Deployment    ${model_pods}  ${exp_replicas}
     ...    ${n_containers}  ${containers}
+
+Wait For ISVC To Be Loaded
+    [Documentation]    Waits for the ISVC to be Loaded (i.e., Success status in the grid)
+    [Arguments]    ${isvc_name}    ${namespace}    ${retries}=36
+    ${rc}    ${isvc_status}=    Run And Return Rc And Output
+    ...    oc get isvc ${isvc_name} -n ${namespace} -o=jsonpath='{.status.modelStatus.states.activeModelState}'
+    ${condition1}=    Evaluate    "${isvc_status}"=="${EMPTY}"
+    ${condition2}=    Evaluate    ${rc}==0
+    ${condition3}=    Evaluate    ${retries} > 0
```

Comment:
Why to have these 4 lines duplicated here before the while branch? Wouldn't be better to delete these 4 lines and put the `Sleep    5s` call at the end of the while loop?

Also, isn't it possible to utilize `oc wait` for this somehow :thinking: (don't insist on this one, but would be even nicer, I believe)

Reply:
I have modified the code in order to not have duplicities.
I'm with you that this one is not an elegant and smart solution, but also I don't think it's possible to combine "oc wait" with the output of a jsonpath.

Reply:
> I have modified the code in order to not have duplicities.

Thank you.

> but also I don't think it's possible to combine "oc wait" with the output of a jsonpath.

After very brief check of oc wait help, something like this could work:
```
oc wait --for=jsonpath='{.status.modelStatus.states.activeModelState}' isvc ${isvc_name} -n ${namespace} --timeout=180s
```

Reply:
@FedeAlonso out of curiosity - this above didn't work?